### DISCUSSION TITLE
Can't push folders containing image files

### DISCUSSION THREAD

**Discussion by mcgowan-family** [community/community#77231] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi there. I am one day in to GitHub and Pages. Firstly, a huge thanks for this resource! 

My project is a simple personal web blog of static html. I've managed to configure the various repo's etc and successfully pushed all my html over which is now live. What I'm really struggling with is getting all the images across. They are 99%  .jpegs, with a few .movs in there. I'm working out of a terminal on a Mac Powerbook, running Git 2.39.2 that I got hold off by downloading the Mac dev toolkit. 

I can push a single jpeg, but when I try and push a folder, it just hangs after the line that starts 'Total'. No error message. I have to Ctl-Z out of it. I've seen comments on forums about Postbuffer and tried that without success. I'd love to hear any ideas :)

**Comment by MantasTech** [comment]
Hi there,

Check File Sizes: GitHub has a 100 MB limit per file. For large files, use [Git LFS](https://git-lfs.github.com/).

Hope that helps!

**Reply by mcgowan-family** [reply]
Thanks @MantasTech . It's a good thought, but nothing is near that size. The image files are generally less than 1Mb. I've been playing around all day with it. The static HTML moves across just fine. I can push a single JPG. But as soon as I try a folder, or nested folder, it looks like it going to work, with the counter racking up[ to 100% on the <push> stats. Then after apparently completing, it just sits there without returning to the command prompt. 

I've seen similar comments in other forums and folk trying changes to Postbuffer size, but that didn't help. Tried garbage collect as well. It feels much more like some kind of constraint around te fact that i'm push a folder of images, but I can't find anything on that anywhere.

I'm sooooo close :)

**Comment by k0290** [comment]
Hi @mcgowan-family 

Have you successfully pushed the changes on another laptop or PC, particularly one with Intel architecture or a VPS? 
What is the size of the folder?

**Reply by mcgowan-family** [reply, marked as answer]
Hi there. Many thanks for reaching out. I've sorted it. The answer appears to have been the postBuffer variable, which I found referenced on some other threads. This was set to 1024 on my system. I set it to 15Mb and everything worked sweet as a nut :) Subsequently I got into further muddles with the head getting ahead of Main in a way I couldn't explain, but I resolved that by simply recloning the repository fro remote onto my system and working off that new local repo, which again feels like a great cure-all. Thanks again for trying to help.
